android/gradle.properties

```diff
@@ -5,5 +5,5 @@
 org.gradle.jvmargs=-Xmx1536M
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableDexingArtifactTransform=false
-shrink=false
\ No newline at end of file
+android.useFullClasspathForDexingTransform=true
```

Comment:
❓ I've only found a StackOverflow [post](https://stackoverflow.com/questions/75847644/android-enabledexingartifacttransform-false-is-deprecated) that suggests this change and also a Unity [issue](https://discussions.unity.com/t/unity-6-android-build-fails-due-to-gradle-version-8-4/359961) that talks about this, but did you find any official documentation that describes this? Also why do we need to set this?

Reply:
I think [this link](https://issuetracker.google.com/issues/225900051) is the most official. It is from Google's issue tracking.

Reply:
Thank you. I followed the "trail of comments" from the link you provided and if I understand correctly, this setting is only needed because of a Gradle bug: https://issuetracker.google.com/issues/230454566

So if I get it right, we will be able to remove this new setting as well once that bug is fixed, so I think we should have a follow-up ticket for this, referencing the Gradle bug ticket so we hopefully won't forget checking it from time to time and removing the setting once it's been fixed. Can you create this ticket @laszlo-domonkos ?

Reply:
Sure thing. I've created this ticket: https://nevissecurity.atlassian.net/browse/NEVISACCESSAPP-6234